src/s3ds.go

```diff
@@ -1,4 +1,4 @@
-package main
+package plugin
 
```

Comment:
src means nothing can you move it to `./plugin` ?

Reply:
Sure! Thanks for your advice! It's done in: [`b8cf0e9`](https://github.com/ipfs/go-ds-s3/pull/275/commits/b8cf0e9c0360c2d72bcabdb95e1cdb05fed611c7)